nailgun/entities.py

```diff
@@ -2666,6 +2666,22 @@ def disable_rhrepo(self, base_arch,
         )
         return _handle_response(response, self._server_config, synchronous)
 
+    def list_available_rhrepos(self, reposet_id):
```

Comment:
Note that at some point in the future I may add a `RepositorySet` entity. At that point, we could replace this method with `RepositorySet.available_repositories`.